doc/0.120/_sources/feeds.rst.txt

```diff
@@ -0,0 +1,169 @@
+.. _feeds:
+
+XML Feeds
+=========
+XML feeds are the way for users to handle repetitive asynchronous bulk uploading of ads.
+When a user is eligible for feed usage the user can configure an HTTP(s) URL through
+:ref:`post_feed_config` or the frontend to make iCAS, once a day, attempt to download an
+XML file. The contents of this XML file need to adhere to the XSD available for sellers
+at :ref:`get_feed_xsd`.
+
+Once a day we pull the XML file configured for all applicable users.
+We treat the provided XML file as the desired list of ads the user wants to have live.
+Any ads not present (referred to by **vendorId**) in the XML file will be paused during
+the import of this file.
+
+The (mandatory) **vendorId** field in the XML file is there to let us know, for consecutive imports, which
+ads are the same. This results in allowing us to update an existing ad with the same **vendorId** instead
+of creating a new ad. **vendorId** is mandatory and, as specified in the XSD, unique for each ad in the
+XML file.
+
+.. warning::
+   There is still an **externalId** field in the XSD, this field is replaced by **vendorId**.
+   Please update your XML to reflect this change. This makes naming consistent between feeds and sellside API.
+   The **vendorId** field in the feeds has the same meaning and constraints as the **vendorId** field in the
+   sellside API.
+
+.. note::
+   If an ad in the feed is unchanged w.r.t. the previous time it was imported, we will skip over this ad and leave
```

Comment:
I think you can omit w.r.t - does not help to read it

Reply:
reworded a bit